src/components/Hero/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Typography from '../Typography';
+import Button from '../Button';
+import HeroBackground from '../Icons/HeroBackground';
+import { Paragraph } from '../../global/commonStyles';
+import { HeroHeader, ButtonWrapper, BackgroundWrapper } from './Hero.style';
+
+const Hero = () => {
+  return (
+    <HeroHeader>
+      <Typography component='h1' headingVariant='hero'>
+        Współpracuj z najlepszym zespołem ekspertów
+      </Typography>
+
+      <Paragraph margin={36} maxWidth={580}>
+        Zadbamy o pozycjonowanie i użyteczność twojego serwisu internetowego.
+        Budujemy wzrost i zwiększamy sprzedaż dla e-commerce, startupów i firm
+        B2B.
+      </Paragraph>
+
+      <ButtonWrapper>
+        <Button fluid>Skontaktuj się</Button>
+        <Button fluid secondary>
+          Learn more
+        </Button>
+      </ButtonWrapper>
+      <BackgroundWrapper>
```

Comment:
To do refactoru z Absolutów i wystarczy wtedy tylko jeden div a raczej nawet od razu SVG